mlflow/models/cli.py

```diff
@@ -35,6 +35,19 @@ def serve(model_uri, port, host, workers, no_conda=False, install_mlflow=False):
     Serve a model saved with MLflow by launching a webserver on the specified host and port. For
     information about the input data formats accepted by the webserver, see the following
     documentation: https://www.mlflow.org/docs/latest/models.html#model-deployment.
+
+    Requests may be made to ``POST /invocations`` in Pandas split- or record-oriented formats.
+
```

Comment:
Prefer active to passive voice and `can` instead of `may`. Thus:

You can make requests to ``POST /invocations`` in Pandas split- or record-oriented formats.